src/test-coverage-checker.php

```diff
@@ -16,6 +16,7 @@
 $percentage = Argument::getPercentage($argv, 2);
 $roundedPrecision = Argument::getInt($argv, 3);
 $shouldExit = Argument::getBool($argv, 4);
+$metricToParse = count($argv) >= 6 && len($argv[5]) > 0 ? Argument::getString($argv, 5) : 'elements';
```

Comment:
I believe the conditional is not needed. You can just parse the 5th argument like all the others, since GitHub will provide the default as 'elements'.

Reply:
Without the conditional, the tests don't pass. To pass the tests, the argument should be mandatory and the rest of tests should be modified, too. What do you prefer?

Reply:
It's fine for the argument to be mandatory, because GitHub will give us the default value (`elements`).

So I think, just like the exit argument, we can set it in the tests as the default value.
And for the tests where we are testing for the metric to parse, we can just override it.

Ex:
```php
protected function runCoverageCheck(string $file, string $percentage = '50', string $roundedPrecision = '2', $metricToParse = 'elements'): string
    {
        $argv = [];
        $argv[1] = __DIR__ . '/Fixtures/' . $file;
        $argv[2] = $percentage;
        $argv[3] = $roundedPrecision;
        $argv[4] = 'false';
        $argv[5] = $metricToParse;
```

So this should make all tests still pass, as it emulates GitHub, and allows us to test for the metric parsing as well if we add the extra parameter in the `runCoverageCheck` calls.

Reply:
Done